client/app/bundles/momentsApp/components/Chart/ChartControl.jsx

```diff
@@ -57,7 +57,7 @@ export default class ChartControl extends React.Component<chartControlProp, char
   render() {
     const { types } = this.props;
     const buttons: any[] = [];
-    _.each(types, (value: string, index: number) => {
+    each(types, (value: string, index: number) => {
```

Comment:
We could completely ditch lodash here and use a for...of loop or forEach.

Reply:
Haha, yah, we could. If we ever need lodash in the future, the babel-plugin-lodash treeshakes the lodash bundle, so only the methods you use gets imported, provided you import them as 
```
import { each } from 'lodash';
```
instead of 
```
import _ from 'lodash'
```

But so far I've only found one usage for lodash. We can remove it for now.

Reply:
That's actually good to know that we have that option.